### PR TITLE
Create metatags for all pages

### PR DESCRIPTION
**Pull request by 7H3XXX**
## What?
Added Metatags and Open graph tags on all pages #1 .

## Why?
This enhances the link sharing experience on social media platforms and SEO : tested. See #1 for more information.

## How?
I'm using the Metadata Object and the `generateMetadata` method from NextJS doc.

## Testing
I used the Social share preview extension (in Chrome) to check the link preview.

### CODE REVIEW

**Comment by 7H3XXX** [comment]
Hello @oktay , did you check the changes I suggested for the #1 issue ? If you haven't can you take some time to review them, it would be nice to add this enhancement to the project.

File: lib/constants.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    title: "Movies",
+    description:
+      "Millions of movies, TV shows and people to discover. Explore now.",
+    url: "http://localhost:3000",
+    siteName: "Movies",
+    images: "/opengraph-image.jpg"
+  },
+};
+
+export const ENV_URL = process.env.ENV === 'dev' ? DEFAULT_URL : "https://movies-flax-sigma.vercel.app"
```
Comment (oktay) [inline lib/constants.ts]: Can we modify the URL to be read within the environment? Let's not hardcode the URL into the project.

`export const DEFAULT_URL = process.env.URL`

Comment (7H3XXX) [reply lib/constants.ts]: My bad, yes, you're right!

Comment (7H3XXX) [reply lib/constants.ts]: I made the necessary changes now, and updated refactor some files to reduce code duplication.
Give it a look and let me know if something's missing.

**Comment by 7H3XXX** [comment]
@oktay  Hi there! How should we proceed on this PR?

**Comment by oktay** [comment]
Thank you for your efforts, but I will address this issue myself with a different approach. I'm closing this pull-request.

**Pull request closed without merging**